docs/source/models/cause.rst

```diff
@@ -4,66 +4,108 @@
+  This decision will be driven by data availability and disease epidemiology. It's 
+  recommended to document any cause with PAF of one relationship with modeled risks 
+  independently. Also, paying attention to situations where you might have multiple causes 
+  in the model with overlapping definitions. For example, diabetes and chronic kidney 
+  disease due to diabetes.
+- **Data Sources:** This section is to find out what data sources are used to inform your
+  cause model and only important to fill out if you deviate from our standard data sources
+  (which will be returned from vivarium_inputs tools).
+- **Component:** This section is primarily necessary for non standard components, and you
+  should be able to answer following questions throughout the filling process.
+    1. What components are used to implement the cause model?
+    2. How do you include them in the simulation?
+    3. What columnsdo they create in the population table?  What do the columns mean?
+    4. What value pipelines do they create? What do the values coming out of 
+       the pipelines mean?
+- **Parameters:** List all available parameters in the format of *parameter name* plus
+  *parameter description*. This section is primarily necessary for non standard 
+  components. It should otherwise point to standard documentation or state that 
+  no parameters are available.
+
 
 Common cause models
 -------------------
 
-.. todo::
-
-   Format as table with model type, description.
-   Fill in descriptions.
+**SI** - Short for susceptible-infected. This modeling strategy is appropriate for 
+causes where people never recover once they enter into a with condition state. 
+Example: Ischemic Heart Disease
 
-SI
-++
+**SIS** - Short for susceptible-infected-susceptible. This modeling strategy 
+is appropriate for causes where individuals can have multiple cases over 
+the course of their lives. Example: Diarrheal Diseases
 
-SIS
-+++
+**SIR** - Short for susceptible-infected-recovered. This modeling strategy
+is appropriate for causes where individuals can only have a single
+case, but do not stay in the with condition state forever. Example: Measles
 
-SIR
-+++
-
-Neonatal
-++++++++
+**Neonatal** - This modeling strategy is appropriate for causes that an 
+individual is born into the with condition state with and never recovers from.
+Example: Preterm birth
 
 Common data sources for cause models
 ------------------------------------
 
-.. todo::
-
-   Format as table with measure, measure definition, data sources and
-   their uses.
-
-Incidence
-+++++++++
-
-Birth prevalence
-++++++++++++++++
-
-Remission
-+++++++++
-
-Prevalence
-++++++++++
-
-Cause-specific mortality
-++++++++++++++++++++++++
-
-Excess mortality
-++++++++++++++++
-
-Disability weight
-+++++++++++++++++
+.. note::
+
+   Currently there are GBD shared function tools that return you draw-level values;
+   alternatively, you can also pull data by using ``vivarium_inputs`` tools.
```

Comment:
we should probably make this an explicit link to our tutorial over there

Reply:
https://vivarium-inputs.readthedocs.io/en/latest/tutorials/pulling_data.html

Reply:
Totally agree, the ultimate goal is to fill the modeling part in details. For example, in the cause modeling we will explicitly explain what's the cause in a way of understanding by GBD, and also what microsim cares about. We will eventually give the handy example how we initialize a simple cause model. Add some workflow images additional to contents in paragraph.